src/pages/patch/patchRestartModal/TaskStatusCheckbox.tsx

```diff
@@ -44,4 +44,7 @@ const StateItemWrapper = styled("div")`
   display: flex;
   flex-direction: row;
   align-items: center;
+  white-space: nowrap;
+  overflow: hidden;
```

Comment:
Does overflow: scroll work? Just in case some one really wanted to view the full task name.

Reply:
After removing this line, I see that it is not necessary to set overflow on the checkbox row because the checkbox container will dynamically adjust scrolling horizontally when the labels are long enough. 
![scrolling gif](https://user-images.githubusercontent.com/10734386/96495762-45872f80-1216-11eb-9443-3ef5d484e3fa.gif)